_posts/2024-06-20-vector-radial-search.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title:  Understanding vector radial search in OpenSearch
+authors:
+  - junqiu
+  - vamshin
+  - dylantong
+date: 2024-06-20
+categories:
+    - technical-posts
+meta_keywords: 
+meta_description: 
```

Comment:
Please update the PR with the following meta: 

_meta_keywords:_ vector radial search, OpenSearch k-NN plugin, k-NN search, neural search

_meta_description:_  Discover how vector radial search enhances the capabilities of the OpenSearch k-NN plugin and increases flexibility and utility in search operations.